src/main/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformer.java

```diff
@@ -122,18 +122,20 @@ public CaseResponse transformExceptionRecord(ExceptionRecord exceptionRecord, bo
             return formTypeValidatorResponse;
         }
 
-        String formType = exceptionRecord.getFormType();
-        log.info("formtype for case {} is {}", caseId, formType);
 
-        if (formType == null || notAValidFormType(formType)) {
-            ScannedData scannedData = sscsJsonExtractor.extractJson(exceptionRecord);
-            String ocrFormType = getField(scannedData.getOcrCaseData(), FORM_TYPE);
+        ScannedData scannedData = sscsJsonExtractor.extractJson(exceptionRecord);
+        String formType = getField(scannedData.getOcrCaseData(), FORM_TYPE);
+
+        if (formType == null  || notAValidFormType(formType)) {
```

Comment:
Your unit tests are only covering 3/4 of the conditions for this.

Reply:
since the ticket mainly mention about to due with form type "Other" case, and i think originally there are already have checking the form type is null unit test. 

Anyway I can add one more for that

Reply:
![image](https://user-images.githubusercontent.com/18538046/185188358-34feb09c-2237-49d5-af30-bf28343fe04e.png)
Still need tests to cover this

Reply:
I tried modified the test, but the validator will block some of those case. so some of condition will not be reach. 
On the other hand, such duplication method will be modify in another ticket. I suggest we can move on, and come back update it later